src/whisper_ctranslate2/whisper_ctranslate2.py

```diff
@@ -330,6 +330,13 @@ def read_command_line():
         default=0.2,
         help="minimum volume threshold to activate listening in live transcribe mode",
     )
+    
+    live_args.add_argument(
+        "--live_device",
```

Comment:
If you can change the name of the parameter and variable across all the code from "live_device" to "live_input_device". In live there is also the notion of device (cpu or gpu), and to make more clear the distinction between both. Thanks